### PR TITLE
feat: adding useIllustration prop to TrustElements

### PR DESCRIPTION
**Pull request by KarlEST**
## 🖼 Context

Our new MCA HP needs `TrustElements` without illustrations so adding a functionality to hide them. 
image

## 🚀 Changes

• Can pass in `useIllustration={false}` to hide the illustration for `TrustElements`
• Add background colour for white illustrations
• Unify TrustElements code example sections
• Move TrustElements to a separate section in docs

## ✅ Checklist

- [x] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [x] Changes are tested and all tests pass
- [x] Changes meet [accessibility standards](https://github.com/transferwise/marketing-components/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [x] Changes work in all supported browsers (don't forget IE11)
- [x] You've updated the documentation if necessary

### CODE REVIEW

File: packages/marketing-components/src/trustelements/TrustElement/TrustElement.js [hunk 1]
```diff
@@ -12,6 +12,14 @@ const TrustElement = ({ title, alt, linkText, href, src, shouldAnimate }) => {
     }
   }
 
+  function getIllustration() {
```
Comment (danlaush) [inline packages/marketing-components/src/trustelements/TrustElement/TrustElement.js:15]: This is a style pref but how would you feel about making this an `<Illustration />` component? Still just defined in this file, nothing major to change, but thematically it keeps the render function as a place for JSX.

``` jsx
const TrustElement = ({shouldAnimate, src, alt, useIllustration}) => {
  const Illustration = () => {
    return shouldAnimate ? (
      <div className="tw-trust-element__svg_container">{src}</div>
    ) : (
      <img className="tw-trust-element__image" src={src} alt={alt} />
    );
  }

  return (
    {useIllustration && <Illustration />}
  )
}
```

Comment (twrdjpalmer) [reply packages/marketing-components/src/trustelements/TrustElement/TrustElement.js]: Should it use props properly (lol): `<Illustration shouldAnimate={shouldAnimate} />`?

Comment (KarlEST) [reply packages/marketing-components/src/trustelements/TrustElement/TrustElement.js]: In that case we should pass  `src` and `alt` as well 🤔

Comment (danlaush) [reply packages/marketing-components/src/trustelements/TrustElement/TrustElement.js]: tyty. Just currently working in an app where everything is `<div>{renderMainPanel()}</div>` like come on we got this toolchain for a reason

**Review by danlaush — APPROVED** [review]
Yeah I like that. I'm surprised it works when the component is defined after it's used since arrow functions aren't hoisted, but if tests pass then it must be fine :D guess it's a babel side effect.

**Review by twrdjpalmer — APPROVED** [review]

**Pull request merged**